### ISSUE TITLE
[macOS Sequoia] Q-Zandronum.app doesn't open, possibly a signature error

### ISSUE THREAD

**Issue by KAMiKAZOW** [IgeNiaI/Q-Zandronum#342]
The usual way of right-click -> Open doesn't work for "Q-Zandronum.app". For Doomseeker it does. Users are not presented with the option to open it anyway. The only way is to manually navigate to System Settings -> Security & Privacy and whitelist the app from there.

I'm not a frequent macOS user but I guess it's a signature error?

macOS Sequoia, Intel

<img width="555" alt="grafik" src="https://github.com/user-attachments/assets/1e3afc70-df4a-4868-9c13-1f88549fcd9b">

**Comment by IgeNiaI** [comment]
Have you run it before on older mac? Maybe they require apps to be signed on latest mac now?

**Comment by KAMiKAZOW** [comment]
> Have you run it before on older mac? Maybe they require apps to be signed on latest mac now?

I didn't before, no. Last weekend on Sequoia was the first time I tried it. As I wrote, the problem isn't present with Doomseeker where "Open Anyway" is possible when using right click to launch the app,

**Comment by IgeNiaI** [comment]
Welp...

**Comment by DragonMinded** [comment]
I'm on Sequoia (15.3) right now and the game launches and runs, although with separate issues I'm still debugging. You might have to pull the quarantine attribute off of the executable with `xattr -d com.apple.quarantine <file>`

**Comment by sidav** [comment]
I've managed to launch it after this:
sudo xattr -cr /Applications/Q-Zandronum.app
codesign --force --deep --sign - /Applications/Q-Zandronum.app
Having no sound whatsoever though.

**Comment by DragonMinded** [comment]
Yeah, I had the same issue. Some sound effects do play, but pretty much all music and most sound effects are missing. There was some repeated errors in the console I could grab next time I'm on my work laptop (was trying to set up a frag night at work and they give us macs).

**Comment by KAMiKAZOW** [comment]
Sound issue is https://github.com/IgeNiaI/Q-Zandronum/issues/343

**Comment by sidav** [comment]
> Sound issue is [#343](https://github.com/IgeNiaI/Q-Zandronum/issues/343)

No, not quite. I do have sound and music if I open Freedoom (without mods) in Q-Zandronum. I don't have neither sound nor music in QCDE. It looks like as if OGG files are not played. Maybe it should be a separate issue.

**Comment by sidav** [comment]
I've created https://github.com/IgeNiaI/Q-Zandronum/issues/352 for sound issues.
